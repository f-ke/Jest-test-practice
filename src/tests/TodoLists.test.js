import React from 'react';
import { shallow } from 'enzyme';
import TodoLists from '../components/TodoLists';
import { textSpanContainsPosition, isExportDeclaration } from 'typescript';
test("test todolist with empty Array",()=>{
    const wrapper = shallow(<TodoLists todos={[]}/>);
    expect(wrapper).toMatchSnapshot();
});
test('should test TodoLists component with list of todos', () => {
    const wrapper = shallow(
     <TodoLists todos={['Read Newspaper', 'Publish Article']} />
    );
    expect(wrapper).toMatchSnapshot();
   });