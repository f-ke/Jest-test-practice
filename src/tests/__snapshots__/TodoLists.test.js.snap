// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test TodoLists component with list of todos 1`] = `
<div
  className="todo-list"
>
  <TodoItem
    key="0"
    todo="Read Newspaper"
  />
  <TodoItem
    key="1"
    todo="Publish Article"
  />
</div>
`;

exports[`test todolist with empty Array 1`] = `
<div
  className="todo-list"
>
  <p>
    No todos found. Please add some.
  </p>
</div>
`;
